### PR TITLE
2.40.0_RN_update

### PR DESCRIPTION
**Pull request by rasika-chivate**
Update the RN for the known issue for PMM 2.40.0.

### CODE REVIEW

File: docs/release-notes/2.40.0.md [hunk 1]
```diff
@@ -110,6 +110,8 @@ VictoriaMetrics has been upgraded [v1.93.4](https://docs.victoriametrics.com/CHA
 - [PMM-12146](https://jira.percona.com/browse/PMM-12146) - Dead Tuples graph on PostgreSQL Vacuum Monitoring Dashboard displayed invalid percentage. The issue has been resolved now.
 - [PMM-12448](https://jira.percona.com/browse/PMM-12448) - Can't start backup if artifact with empty `service_id` exists.
 
-## Known issue
+## Known issues
 
 - [PMM-12517](https://jira.percona.com/browse/PMM-12517) - If you have set custom dashboards as your Home dashboard in version 2.26.0, they may be lost after upgrading to version 2.40.0. To prevent this, we recommend that you back up your custom dashboard before upgrading and recover it after the upgrade.
+
+- [PMM-12592](https://jira.percona.com/browse/PMM-12592) - We do not recommend updating to version 2.40.0 due to a critical issue during the migration from SQLite to PostgreSQL. Instead, we suggest waiting for version PMM 2.40.1 to be released.
```
Comment (BupycHuk) [inline docs/release-notes/2.40.0.md:117]: can we make it big bold warning in the beginning of RN?

Comment (rasika-chivate) [reply docs/release-notes/2.40.0.md:117]: ok

**Review by rnovikovP — CHANGES_REQUESTED** [review]
This must to be positioned from the user problem, something like: "If you are using Grafana Dashboards Library Panel (name needs to be correct) - don't upgrade, as we found a bug. Wait for 2.40.1."
No one is interested in the reason for this (SQLite->PG migration ), as this is internal stuff. There is no user-faced feature of pmm called "SQLite/PG database inside PMM."

**Comment by rasika-chivate** [comment]
> f you are using Grafana Dashboards Library Panel (name needs to be correct) - don't upgrade, as we found a bug. Wait for 2.40.1." No one is interested in the reason for this (SQLite->PG migration ), as this is internal stuff. There is no user-faced feature of pmm called "SQLite/PG database inside PMM."

Fixed.

**Review by rnovikovP — APPROVED** [review]

**Pull request merged**